google/ggrc-core
# GGRC-8250 Replace Deferred with Promise part 6

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] GGRC-8246

# Issue description

In scope of the migration to CanJS 3 also jQuery was updated to version 3. In version 3 some async changes were introduced for Deferred. It allows to start replacement for Deferred to Promise usages.

# Steps to test the changes

1. tree-view-utils
1.1 Open any object or page.
1.2 Expected: It works.
2. snapshots-util
2.1 Open any object with snapshotable objects.
2.2 Expected: Counts of this snapshotable objects displays correctly.
3. info-pin-controller.
3.1 Open info-pane of any object.
3.2 Expected: It works.
4. repeat-on-button-wrapper
4.1 Start creating workflow.
4.2 Click 'Repeat off' button and add repeating.
4.3 Expected: It works.

# Solution description

Replace Deferred with Promise.

# Sanity checklist




# PR Review checklist
